app/controllers/dashboard_controller.rb

```diff
@@ -202,13 +202,20 @@ def landing_page
   def new_landing_page
     @search_results = search_results(params[:search])
 
-    @org_owners = User.findaproject_orgs.order(:display_name).joins(:collections).left_outer_joins(:document_sets).distinct
-    @individual_owners = User.findaproject_individuals.order(:display_name).left_outer_joins(:document_sets).joins(:collections).distinct
-    
-
-    # Sampled Randomly down to 8 items for Carousel
-    docsets = DocumentSet.carousel.includes(:owner).where(owner_user_id: @org_owners.ids + @individual_owners.ids).sample(5)
-    colls = Collection.carousel.includes(:owner).where(owner_user_id: @org_owners.ids + @individual_owners.ids).sample(5)
+    users = User.owners
+                .joins(:collections)
+                .left_outer_joins(:document_sets)
+                .includes({ random_collections: :tags }, { random_document_sets: { collection: :tags } })
+
+    @org_owners = users.findaproject_orgs.order(:display_name).distinct
+    @individual_owners = users.findaproject_individuals.order(:display_name).distinct
+
+    docsets = DocumentSet.carousel
+                         .includes(:owner, { next_untranscribed_page: :work })
+                         .where(owner_user_id: @org_owners.select(:id) + @individual_owners.select(:id)).sample(5)
+    colls = Collection.carousel
+                      .includes(:owner, { next_untranscribed_page: :work })
```

Comment:
N+1 issue here. Checking the html we access next_untranscribed_page, and then it's work. Both of these have to be preloaded.

Also it seems, as long as we preload correct relationships, turning it to array afterwards will not cause more query

Reply:
I believe that the point of the join to `next_untranscribed_page` and `work` is to create a link to transcribe the page.  There might be alternative ways to accomplish this without joining to the `works` table, perhaps through a new route that just takes `page_id`, or `page_id` and collection/owner info.